integration_tests/structs_14.py

```diff
@@ -1,32 +1,72 @@
-from ltypes import i8, dataclass, i32, ccallable
-from numpy import empty, int8
+from ltypes import i8, dataclass, i32, f32, c32, f64, ccallable
+from numpy import empty, int8, int32, float32, complex64, float64
 from copy import deepcopy
 
 @dataclass
 class buffer_struct:
     buffer: i8[32]
+    buffer1: i32[32]
+    buffer2: f32[32]
+    buffer3: c32[32]
+    buffer4: f64[32]
 
 @ccallable
 @dataclass
 class buffer_struct_clink:
     buffer: i8[32]
+    buffer1: i32[32]
+    buffer2: f32[32]
+    buffer3: c32[32]
+    buffer4: f64[32]
```

Comment:
Let's do also i16 and i64, those are commonly used. Let's do c64 too, might as well. Then everything is covered.